### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Fix linter warnings

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Review by alexAubin — APPROVED** [review]
LGTM

**Comment by alexAubin** [comment]
Restarted the test because the CI was improved since then, let's see if we can track the 502 error

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by alexAubin** [comment]
So zblerg, not sure why exactly it's failing, naively I see this :  

```
Aug 02 13:43:06 yarn[7833]: 2021-08-02 13:43:06 [INFO] The Lounge v4.2.0 (Node.js 14.17.4 on linux x64)
Aug 02 13:43:06 yarn[7833]: 2021-08-02 13:43:06 [INFO] Configuration file: /home/yunohost.app/thelounge/config.js
Aug 02 13:43:06 yarn[7833]: 2021-08-02 13:43:06 [INFO] Available at http://[::]:9000/ in private mode
Aug 02 13:43:06 yarn[7833]: 2021-08-02 13:43:06 [INFO] New VAPID key pair has been generated for use with push subscription.
Aug 02 13:43:06 yarn[7833]: 2021-08-02 13:43:06 [INFO] There are currently no users. Create one with thelounge add <name>.
Aug 02 13:43:13 yarn[7833]: 2021-08-02 13:43:13 [INFO] Exiting...
```

So maybe we gotta add some `thelounge add <somename>` ?

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: conf/systemd.service [hunk 1]
```diff
@@ -6,12 +6,15 @@ After=network.target
...
 Environment="NODE_ENV=production"
 ExecStart=/usr/bin/yarn start
-Restart=always
+Restart=on-failure
+RestartSec=5
+StartLimitInterval=60s
+StartLimitBurst=3
 
 [Install]
-WantedBy=default.target
```
Comment (ericgaspar) [inline conf/systemd.service:17]: @alexAubin The problem was this line. (It took me so long to figure this out...🙄)

**Pull request merged**